astropy/convolution/kernels.py

```diff
@@ -146,10 +146,12 @@ class Gaussian2DKernel(Kernel2D):
     _separable = True
     _is_bool = False
 
-    def __init__(self, stddev, **kwargs):
-        self._model = models.Gaussian2D(1. / (2 * np.pi * stddev ** 2), 0,
-                                        0, stddev, stddev)
-        self._default_size = _round_up_to_odd_integer(8 * stddev)
+   def __init__(self, width, height, position_angle, support_scaling=8, **kwargs):
```

Comment:
I think the parameter names should probably be the same as for ``Gaussian2D`` for consistency - also, the position angle and height should be optional (and the height should default to the width) to preserve backward-compatibility and ease-of-use